js/core/index.js

```diff
@@ -167,6 +167,9 @@ require([
       .success(function(data) {
         config.cluster.infos = data;
         $(document).trigger('show', { page: config.STARTPAGE });
+      })
```

Comment:
it fixes bug on login page refresh when accessing on another cluster where user is not yet authenticated: this page is not updated otherwise